### PR TITLE
Patch 1 added apps and latest launch app code

### PR DESCRIPTION
**Pull request by mannyman**

### CODE REVIEW

File: hisensetv/__main__.py [hunk 1]
```diff
@@ -51,6 +51,22 @@ def main():
...
+            "vudu",
+            "hbo_max",
+            "youtube_music",
+            "iheartradio",
+            "pandora",
+            "tidal",
+            "spotify",
+            "tunein_radio",
+            "plex",
+            "mrmc",
```
Comment (newAM) [inline hisensetv/__main__.py]:
```suggestion
            "mrmc",
```

There's a trailing space there.

**Review by newAM — CHANGES_REQUESTED** [review]
Looks good to me - but it looks like this is missing the JSON payload changes to `__init__.py` described in #19 , let me know if you need help with git stuff!

File: hisensetv/__main__.py (hunk 1 above)
Comment (mannyman) [reply hisensetv/__main__.py]: Thanks.
have no idea how to use GIT, so used notepad++ to find all the spaces and tabs and linefeeds I put in by accident.
I then copy/pasted what worked at home over.

**Comment by mannyman** [comment]
The init stuff is here. No idea what to do next.
https://github.com/mannyman/hisensetv/blob/mannyman-patch-1/hisensetv/__init__.py

**Comment by newAM** [comment]
I added those changes to this pull request, let me know if it looks good to you and I'll merge!

**Comment by mannyman** [comment]
Sounds good. I guess after it's merged I reinstall and see everything still works. But I have a question.
Do the changes to make it work for my TV screw it up for those with older TVs?

**Comment by newAM** [comment]
> Do the changes to make it work for my TV screw it up for those with older TVs?

I do not have any way of testing, my guess would be that the API updates with the TV.
If someone else opens an issue about it then I'll revert it to the old style.

**Pull request merged**